dashboard/app/models/blockly.rb

```diff
@@ -22,6 +22,7 @@ class Blockly < Level
     default_num_example_blocks
     open_function_definition
     callout_json
+    project_template_level_name
```

Comment:
One concern: this is creating an association based on the name of the target level, not the ID.  If the name of that level ever changes it will break the association.